.github/workflows/eslint-config-custom-publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Eslint Config Publish
```

Comment:
looks like something we will need in every published package, can we reuse it somehow?

Reply:
The name? Why we would like to reuse the name?

Reply:
He means the whole file, most of it is the same in most GitHub actions

Reply:
what about this? do you want us to maintain this config for each package? this sounds like a bad idea

Reply:
seems like its possibe to use `uses` and `using`? https://stackoverflow.com/questions/65532726/github-actions-extend-existing-job

Reply:
Figured out how it can be used.